_docker/drupal/drupal-filesystem/web/themes/custom/rhdp/rhdp.theme

```diff
@@ -416,6 +416,50 @@ function rhdp_preprocess_node(array &$variables) {
     }
   }
 
+<<<<<<< HEAD
```

Comment:
@visuale Looks like you've got git merge resolution in the actual commit here. This is going to cause PHP to error cc. @jordanpagewhite

Reply:
Ah, thanks for noticing that and pointing it out Rob. I just glanced at this and thought it might still be a broken build from the issues we had with the export process.